SETUP/SETUP.BAT

```diff
@@ -518,6 +518,8 @@ vdelay 1000
 
 :PartitionManual
 vcls /g /fGray /bBlack
+echo NOTICE: FAT16 partition under 2Gb is needed for KERNL86.   
+vdelay 2000

```

Comment:
This is fine and can accept the vdelay as-is. However, I would like you to consider the following instead. 

```
vecho /g /n /t %LANGFILE% PRESS_KEY
vpause /t 5
```

Instead of a mandatory 2 second delay, the user is presented with a "Press a key to continue." message in their own language. The pause automatically times out after 5 seconds if the user does nothing.